src/compare.py

```diff
@@ -126,6 +133,59 @@ def compare_phash(source: MatLike, capture: MatLike, mask: MatLike | None = None
     return 1 - (hash_diff / 64.0)
 
 
+# copied from https://github.com/madmaze/pytesseract
+def subprocess_args():
```

Comment:
I'd move this to `utils.py` and include all relevant comments from the recipe.
The top comment in the recipe should be included as a docstring. The link to the recipe should also be included in the docstring.